consensus/istanbul/backend/announce.go

```diff
@@ -190,6 +253,8 @@ func (sb *Backend) handleIstAnnounce(payload []byte) error {
 		return err
 	}
 
+	log.Error("Handling an IstanbulAnnounce message", "actual", msg.IncompleteEnodeURL)
```

Comment:
nit:  Please change this severity to Debug.

Reply:
I think this is a debug message i forgot to remove, will remove